Jucer2CMake/src/reprojucer.hpp

```diff
@@ -431,6 +443,17 @@ inline void writeReprojucerCMakeLists(const Arguments& args,
       writeQuoted(cmakeKeyword, value);
     };
 
+  const auto convertRawStringSetting =
```

Comment:
The `convert*Setting` lambdas also have a specific order. Please put `convertRawStringSetting` and `convertRawStringSettingIfDefined` between `convertOnOffSettingWithDefault` and `convertSettingAsList`.

Reply:
Addressed in 22df5395c361c114111f985a2172a8ace1563de4.